shipper.py

```diff
@@ -86,8 +86,84 @@ def shipment_get_all():
         return json.dumps({'error': 'bad request'})
 
 
+@app.route('/api/v1/shipment/<shipmentid>/status', method='GET')
+def shipment_get_status(shipmentid):
+    try:
+        data = db['shipments'].find_one({'id': shipmentid})
+        if data is not None:
+            if 'status' in data:
+                response.status = 200
+                return {'id': shipmentid, 'status': str(data['status'])}
+            else:
+                response.status = 400
+                return {'error': 'shipment data incomplete'}
+    except:
+        raise
```

Comment:
Is the execption raised here actually encapsulated in a JSON document? Just wondering.